src/parser/alter.rs

```diff
@@ -99,6 +99,47 @@ impl Parser<'_> {
         }
     }
 
+    /// Parse ALTER CONNECTOR statement
+    /// ```sql
+    /// ALTER CONNECTOR connector_name SET DCPROPERTIES(property_name=property_value, ...);
+    /// or
+    /// ALTER CONNECTOR connector_name SET URL new_url;
+    /// or
+    /// ALTER CONNECTOR connector_name SET OWNER [USER|ROLE] user_or_role;
+    /// ```
+    pub fn parse_alter_connector(&mut self) -> Result<Statement, ParserError> {
+        let name = self.parse_identifier()?;
+        self.expect_keyword_is(Keyword::SET)?;
+
+        let properties = match self.parse_options_with_keywords(&[Keyword::DCPROPERTIES]) {
```

Comment:
```suggestion
        let properties = match self.parse_options_with_keywords(&[Keyword::DCPROPERTIES])? {
```
this seems to ignore potential errors?

Reply:
Done!